docs/api/cypress-api/ensure.mdx

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Can we break these docs into their own PR?

Reply:
https://github.com/cypress-io/cypress-documentation/pull/5054